pkg/security/telemetry/telemetry.go

```diff
@@ -49,7 +50,13 @@ func (c *ContainersTelemetry) ReportContainers(metricName string) {
 			}
 		}
 
-		c.TelemetrySender.Gauge(metricName, 1.0, []string{"container_id:" + container.ID})
+		var mode string
+		if os.Getenv("ECS_FARGATE") == "true" || os.Getenv("DD_ECS_FARGATE") == "true" {
+			mode = "fargate_ecs"
+		} else if os.Getenv("DD_EKS_FARGATE") == "true" {
+			mode = "fargate_eks"
+		}
+		c.TelemetrySender.Gauge(metricName, 1.0, []string{"container_id:" + container.ID, "mode:" + mode})
```

Comment:
Do we want to include `mode:""` if not on fargate ?

Reply:
going to add default